hazelcast/src/main/java/com/hazelcast/internal/management/dto/SlowOperationDTO.java

```diff
@@ -0,0 +1,74 @@
+package com.hazelcast.internal.management.dto;
+
+import com.eclipsesource.json.JsonArray;
+import com.eclipsesource.json.JsonObject;
+import com.eclipsesource.json.JsonValue;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.hazelcast.util.JsonUtil.getArray;
+import static com.hazelcast.util.JsonUtil.getInt;
+import static com.hazelcast.util.JsonUtil.getString;
+
+/**
+ * A Serializable DTO for {@link com.hazelcast.spi.impl.operationexecutor.slowoperationdetector.SlowOperationLog}.
+ */
+public class SlowOperationDTO {
```

Comment:
Most (all?) of the DTO's in the managent.dto package implement JSonSerializable. So would make sure that your code is in line.